cmd/collectors/rest/rest.go

```diff
@@ -169,15 +182,7 @@ func (r *Rest) getClient(a *collector.AbstractCollector, config *node.Node) (*re
 		r.Logger.Error().Str("poller", opt.Poller).Msg("Address is empty")
 		return nil, errs.New(errs.ErrMissingParam, "addr")
 	}
-
-	clientTimeout := config.GetChildContentS("client_timeout")
-	timeout := rest.DefaultTimeout * time.Second
-	duration, err := time.ParseDuration(clientTimeout)
-	if err == nil {
-		timeout = duration
```

Comment:
changes are fine, but could be possible here as well without much modification:
 r.Client.Timeout = timeout